test/remote/device_tests.js

```diff
@@ -376,9 +376,47 @@ describe('remote device', function () {
   )
 
   it(
-    'update device works with callback urls that have ports',
+    'update device works with callback urls that :443 as a port',
     () => {
-      var goodPushCallback = 'https://updates.push.services.mozilla.com:433'
+      var goodPushCallback = 'https://updates.push.services.mozilla.com:443'
```

Comment:
Should we add a test w/ a trailing slash too?

Reply:
eh, might be too much

Reply:
let's use this real callback here:

```
https://updates.push.services.mozilla.com:443/wpush/v1/gAAAAABbkq0Eafe6IANS4OV3pmoQ5Z8AhqFSGKtozz5FIvu0CfrTGmcv07CYziPaysTv_9dgisB0yr3UjEIlGEyoprRFX1WU5VA4nG-9tofPdA3FYREPf6xh3JL1qBhTa9mEFS2dSn--
```